MediaStorageSample/README.md

```diff
@@ -1,15 +1,18 @@
 # Media Storage Sample
 * In this sample, you can list your Media IDs.
 
-## Setup
-* Complete [Setup](https://github.com/ricohapi/media-storage-swift#setup) of Media Storage for Swift.
+## Using the Sample
+* Clone Ricoh Media Storage for Swift by running the following commands:
+```sh
+$ git clone --recursive https://github.com/ricohapi/MediaStorage.git
+```
+* Open `MediaStorage.xcodeproj` in the `MediaStorage` directory, and open `MediaStorage.xcodeproj`.
```

Comment:
I think last "and open `MediaStorage.xcodeproj`" sentence is unnecessary because it duplicated to first "Open `MediaStorage.xcodeproj`".